physical/optimizer/scenarios.go

```diff
@@ -12,6 +12,7 @@ import (
 
 var DefaultScenarios = []Scenario{
 	MergeRequalifiers,
+	RemoveEmptyMaps,
```

Comment:
Please keep _some_ _semblance_ of order here. And don't stick a Remove between two Merge'es :P